server/src/routes/api/v1/pizzaPlaceRouter.js

```diff
@@ -0,0 +1,17 @@
+import express from "express"
+import { PizzaPlace } from "../../../models/index.js"
+import PizzaPlaceSerializer from "../../../serializers/PizzaPlaceSerializer.js"
+
+const pizzaPlaceRouter = new express.Router()
+
+pizzaPlaceRouter.get("/", async (req, res) => {
+    try {
+        const pizzaPlaces = await PizzaPlace.query()
+        const serializedPizzaPlaces = pizzaPlaces.map(pizzaPlace => PizzaPlaceSerializer.getSummary(pizzaPlace))
```

Comment:
Ideally the work of the `.map` is done in the serializer so that the bulk of the work is done there, rather than here in the router. We generally set up serializers with 2 different methods (feel free to name the methods what you want)
- `getSummary` intended to accept an array of objects, and will `map` over each one to serialize
- `getDetails` intended to accept ONE object as an argument, and serializes that one object